oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/ElasticsearchIndexConstants.java

```diff
@@ -1,21 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.jackrabbit.oak.plugins.index.elasticsearch;
-
-public class ElasticsearchIndexConstants {
-    public static final String TYPE_ELASTICSEARCH = "elasticsearch";
```

Comment:
It might make sense to keep this file for constants (since these would get used a lot in tests even outside the elastic module) - I think Using ElasticsearchIndexConstants.TYPE_ELASTICSEARCH sounds better than using ElasticsearchIndexDefinition.TYPE_ELASTICSEARCH

Reply:
Please see my comment here https://github.com/oak-indexing/jackrabbit-oak/pull/94#discussion_r400836707